shared/domain/src/commonMain/kotlin/network/bisq/mobile/domain/service/network/KmpTorService.kt

```
@@ -336,16 +335,14 @@ class KmpTorService : BaseService(), Logging {
             repeat(3) { attempt ->
                 try {
-                    Socket().use { socket ->
-                        socket.connect(InetSocketAddress("127.0.0.1", controlPort), 1000)
-                        log.i { "Verified control port $controlPort is accessible" }
-                        return
-                    }
+                    val socket = aSocket(selectorManager).tcp().connect("127.0.0.1", controlPort)
+                    socket.close()
+                    log.i { "Verified control port $controlPort is accessible" }
+                    return
                 } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)